zuul.d/jobs.yaml

```diff
@@ -45,3 +45,9 @@
     parent: awx-tests
     vars:
       command: /start_tests.sh detect-schema-change
+
+- job:
+    name: awx-ansible-modules
+    parent: awx-tests
+    vars:
+      command: /start_tests.sh prepare_test_modules
```

Comment:
@Spredzy After discussion in the related pull request with @elyezer https://github.com/ansible/awx/pull/4701#discussion_r324227177 we want to move toward a different name, `test_modules_all`.

Sorry about the churn.

Reply:
I guess it would be. The wording of "tower modules" is largely historical.

Reply:
`test_collection_all`